adapters/nextmillennium/nextmillennium.go

```diff
@@ -98,12 +86,16 @@ func (adapter *adapter) buildAdapterRequest(prebidBidRequest *openrtb2.BidReques
 		Headers: headers}, nil
 }
 
-func createBidRequest(prebidBidRequest *openrtb2.BidRequest, params *openrtb_ext.ImpExtNextMillennium) *NextMillenniumBidRequest {
+type nextMillJsonExt struct {
+	Prebid struct {
+		Storedrequest struct {
+			ID string `json:"id"`
+		} `json:"storedrequest"`
+	} `json:"prebid"`
+}
+
```

Comment:
I'm confused why this object is written in this way with nested structs? I think I understand that these structs are necessary for the json marshalling, but could this `nextMillJsonExt` be re-written, or these structs be written separately to improve readability?

Reply:
Done,
Please confirm if it's looks good.

Thanks